### PR TITLE
Update content on how to extend Cypher

### PR DESCRIPTION
**Pull request by NataliaIvakina**
Update the page according to the syntax of Neo4j 5.x.
Update information about APOC Core library.
Add the link to the Cypher Cheat Sheet.

### CODE REVIEW

File: modules/ROOT/pages/cypher-intro/procedures-functions.adoc [hunk 1]
```diff
@@ -4,126 +4,107 @@
...
 But sometimes you need to do more than it currently offers, like additional graph algorithms, parallelization, or custom conversions.
 
-That's why Neo4j and Cypher can be extended with link:https://neo4j.com/docs/java-reference/current/extending-neo4j/[_User-defined procedures and functions_].
+Cypher can be extended with _User-defined procedures and functions_, as described in link:https://neo4j.com/docs/java-reference/current/extending-neo4j/procedures/[Java Reference -> User-defined procedures] and link:https://neo4j.com/docs/java-reference/current/extending-neo4j/functions/[Java Reference -> User-defined functions]. 
+
 Neo4j itself provides and utilizes custom procedures.
-Many of the monitoring, introspection and security features exposed by Neo4j Browser are implemented using procedures.
+Many of the monitoring, introspection and security features exposed by *Neo4j Browser* are implemented using procedures.
 
 image::procedures-functions-bolt.jpg[width=600]
```
Comment (NataliaIvakina) [inline modules/ROOT/pages/cypher-intro/procedures-functions.adoc:26]: I have a question on this image. Do we really need it here? Does it add any new information or provide any clarification?

Comment (jexp) [reply modules/ROOT/pages/cypher-intro/procedures-functions.adoc:26]: It was meant to illustrate the relationship between drivers, cypher and user defined functions/procs being part of the server and embedded in Cypher during execution.

File: modules/ROOT/pages/cypher-intro/procedures-functions.adoc [hunk 2]
```diff
@@ -4,126 +4,107 @@
...
 
 [#procedures-functions]
-== What are procedures and functions?
+== Procedures and functions in Neo4j
 
 * Functions are simple computations / conversions and return a single value.
 * Functions can be used in any expression or predicate.
 
 * Procedures are more complex operations and generate streams of results.
-* Procedures must be used within the `CALL` clause and `YIELD` their result columns.
```
Comment (jexp) [inline modules/ROOT/pages/cypher-intro/procedures-functions.adoc:36]: we should keep this important difference to functions.

File: modules/ROOT/pages/cypher-intro/procedures-functions.adoc [hunk 3]
```diff
@@ -4,126 +4,107 @@
...
-* They can generate, fetch or compute data to make it available to later processing steps in your Cypher query.
+* To check which procedures are available in your Neo4j DBMS, use the Cypher command `SHOW PROCEDURES`.
+* Procedures can generate, fetch, or compute data to make it available to later processing steps in your Cypher query.
 
 [#cypher-list-extension]
-== Listing & using functions and procedures
+== Listing and using functions and procedures in Neo4j
 
-There are a link:https://neo4j.com/docs/operations-manual/current/reference/procedures/[number of built in procedures^], two of which are used to list available functions and procedures.
+There are a number of built-in procedures in Neo4j.
```
Comment (jexp) [inline modules/ROOT/pages/cypher-intro/procedures-functions.adoc:41]: The language here feels less fluent, you could actually inline the manual link into the first bullet.

File: modules/ROOT/pages/cypher-intro/procedures-functions.adoc [hunk 4]
```diff
@@ -4,126 +4,107 @@
...
-----
-CALL dbms.procedures()
-----
+* `SHOW FUNCTIONS`
+* `SHOW PROCEDURES`
+
+You can refer to the link:https://neo4j.com/docs/cypher-cheat-sheet/current/[Cypher Cheat Sheet] to get a quick reference on how to use these commands.
 
-Call `dbms.functions()` to list functions.
+Also take a look at the link:https://neo4j.com/docs/java-reference/current/extending-neo4j/[procedure section in the Neo4j Java Reference^].
```
Comment (jexp) [inline modules/ROOT/pages/cypher-intro/procedures-functions.adoc:51]: we should think about how often we need to reference the same links on the same page while still adding value for the readers.

File: modules/ROOT/pages/cypher-intro/procedures-functions.adoc [hunk 5]
```diff
@@ -4,126 +4,107 @@
...
 
 Each procedure returns one or more columns of data.
-With `yield` these columns can be selected and also aliased and are then available in your Cypher statement.
+With `YIELD` clause these columns can be selected and also aliased and are then available in your Cypher statement.
```
Comment (jexp) [inline modules/ROOT/pages/cypher-intro/procedures-functions.adoc:54]: With _the_

File: modules/ROOT/pages/cypher-intro/procedures-functions.adoc [hunk 6]
```diff
@@ -4,126 +4,107 @@
...
-CALL dbms.procedures()
+SHOW PROCEDURES
 YIELD name, signature, description as text
 WHERE name STARTS WITH 'db.'
 RETURN * ORDER BY name ASC
 ----
 
-Of course you can also process the result columns with other Cypher clauses.
-Here we group them by package.
+Of course, you can process the result columns with other Cypher clauses and group the results.
```
Comment (jexp) [inline modules/ROOT/pages/cypher-intro/procedures-functions.adoc:64]: This is not fully longer true. 
A better wording would be: "Like other Cypher admin commands `SHOW PROCEDURES` can be used with a subset of Cypher clauses, as shown above where we filter by `'db.'` prefix and return the results ordered by name.

File: modules/ROOT/pages/cypher-intro/procedures-functions.adoc [hunk 7]
```diff
@@ -4,126 +4,107 @@
...
 
-[source,cypher]
```
Comment (jexp) [inline modules/ROOT/pages/cypher-intro/procedures-functions.adoc:67]: With the unfortunate limitations of Admin commands, one would have to rewrite the statement below to:
Which is not as readable anymore.

```
SHOW PROCEDURES
YIELD name, signature, description
RETURN split(name,".")[0..-1] AS package,  count(*) AS count,
       collect(split(name,".")[-1]) AS names
ORDER BY count DESC
```

File: modules/ROOT/pages/cypher-intro/procedures-functions.adoc [hunk 8]
```diff
@@ -4,126 +4,107 @@
...
-|["db"]              |9      |["awaitIndex","constraints","indexes","labels",..,"schema"]
-|["dbms"]            |9      |["components","functions","queries","procedures",...]
-|===
+* user-defined scalar functions,
+* user-defined aggregation functions.
 
-As of Neo4j 3.1, all functions available are directly part of the Cypher implementation, so user-defined Functions would only come from installed libraries.
+For more details, see link:https://neo4j.com/docs/cypher-manual/current/functions/user-defined/[Cypher Manual -> User-defined functions].
 
 You can take any procedure library and deploy it to your server to make additional procedures and functions available.
```
Comment (jexp) [inline modules/ROOT/pages/cypher-intro/procedures-functions.adoc:74]: to your _self managed_ server.

NOTE: In Neo4j AuraDB the set of available procedures and functions is limited to the built-in ones and a subset of the https://neo4j.com/docs/aura/platform/apoc/[APOC Core library^].

File: modules/ROOT/pages/cypher-intro/procedures-functions.adoc (hunk 4 above)
Comment (jexp) [reply modules/ROOT/pages/cypher-intro/procedures-functions.adoc:51]: I think we can leave this one off as it was mentioned before and comes again later.

File: modules/ROOT/pages/cypher-intro/procedures-functions.adoc [hunk 9]
```diff
@@ -4,126 +4,107 @@
...
 
 [#deploy-extension]
-== Deploying procedures & functions
+== Deploying procedures and functions
 
-If you built your own procedures or downloaded them from a community project, they are packaged in a jar-file.
+If you build your own procedures or download them from a community project, they are packaged in a JAR file.
 You can copy that file into the `$NEO4J_HOME/plugins` directory of your Neo4j server and restart.
 
+You can take any procedure library and deploy it to your server to make additional procedures and functions available.
```
Comment (jexp) [inline modules/ROOT/pages/cypher-intro/procedures-functions.adoc:84]: Duplication of the sentence above? Either remove it there or here.

File: modules/ROOT/pages/cypher-intro/procedures-functions.adoc (hunk 8 above)
Comment (jexp) [reply modules/ROOT/pages/cypher-intro/procedures-functions.adoc:74]: Move this NOTE and the previous down to the box with Aura

File: modules/ROOT/pages/cypher-intro/procedures-functions.adoc [hunk 10]
```diff
@@ -153,13 +135,32 @@ RETURN apoc.date.format(timestamp()) as time,
...
+* database/api integration
+* graph refactorings
+* import and export
+* spatial index lookup
+* rdf import and export
+* and many more
+
+[NOTE]
+====
+Community projects are not supported officially and we don’t provide any SLAs or guarantees around backwards compatibility and deprecation.
```
Comment (jexp) [inline modules/ROOT/pages/cypher-intro/procedures-functions.adoc:152]: Community and Neo4j Labs projects are ...

**Review by jexp — APPROVED** [review]
Please see my comments.

File: modules/ROOT/pages/cypher-intro/procedures-functions.adoc (hunk 7 above)
Comment (NataliaIvakina) [reply modules/ROOT/pages/cypher-intro/procedures-functions.adoc:67]: Thank you, Michael. I tried to rewrite this query, but my attempts were not successful.

**Pull request merged**